app/models/lock.rb

```diff
@@ -37,7 +37,7 @@ def reason
   end
 
   def self.remove_expired_locks
-    Lock.where("delete_at IS NOT NULL and delete_at < CURRENT_TIMESTAMP").find_each(&:soft_delete!)
+    Lock.where("delete_at IS NOT NULL and delete_at < ?", Time.now.to_s(:db)).find_each(&:soft_delete!)
```

Comment:
I've reverted it as the CURRENT_TIMESTAMP is working fine again on our local machines